h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+
+import CreateGroupForm from '../CreateGroupForm';
+
+describe('CreateGroupForm', () => {
+  const createWrapper = () => mount(<CreateGroupForm />);
+
+  [
+    { counter: 'name', limit: 25 },
+    { counter: 'description', limit: 250 },
+  ].forEach(({ counter, limit }) => {
```

Comment:
There are two character counters and I want to have the same test(s) for each. Doing the equivalent of `parametrize` in pytest, not sure if you have an established pattern for this in JavaScript, but this is the same as the "dynamically generating tests" pattern in the Mocha docs: https://mochajs.org/#dynamically-generating-tests

Reply:
This is exactly what we do in other projects. We did have a helper for this that we used at one point, but we stopped using it as it didn't add much value over just what you've done here.

Reply:
:+1: